### PR TITLE
Add Swagger documentation to data plane Enterprise Knowledge Graph

### PR DESCRIPTION
**Pull request by tengr**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/EnterpriseKnowledgeGraph/data-plane/Microsoft.EnterpriseKnowledgeGraph/stable/2018-12-03/searchswagger.json [hunk 1]
```diff
@@ -0,0 +1,960 @@
+{
+  "openapi": "3.0.1",
```
Comment (nschonni) [inline specification/EnterpriseKnowledgeGraph/data-plane/Microsoft.EnterpriseKnowledgeGraph/stable/2018-12-03/searchswagger.json]: Don't think 3 is supported

**Comment by KrisBash** [comment]
This is data-plane, no ARM review required

File: specification/EnterpriseKnowledgeGraph/data-plane/Microsoft.EnterpriseKnowledgeGraph/stable/2018-12-03/searchswagger.json [hunk 2]
```diff
@@ -0,0 +1,710 @@
...
+          },
+          "readOnly": true,
+          "type": "object"
+        },
+        "startTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "status": {
+          "enum": [
```
Comment (nschonni) [inline specification/EnterpriseKnowledgeGraph/data-plane/Microsoft.EnterpriseKnowledgeGraph/stable/2018-12-03/searchswagger.json]: min/max instead on enum?

File: specification/EnterpriseKnowledgeGraph/data-plane/Microsoft.EnterpriseKnowledgeGraph/stable/2018-12-03/searchswagger.json [hunk 3]
```diff
@@ -0,0 +1,710 @@
...
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/EntityBaseResult"
+            }
+          }
+        },
+        "tags": [
+          "Entities"
+        ],
+        "operationId": "ReadRawEntitesAsync"
```
Comment (nschonni) [inline specification/EnterpriseKnowledgeGraph/data-plane/Microsoft.EnterpriseKnowledgeGraph/stable/2018-12-03/searchswagger.json]:
```suggestion
        "operationId": "ReadRawEntitiesAsync"
```

File: specification/EnterpriseKnowledgeGraph/data-plane/Microsoft.EnterpriseKnowledgeGraph/stable/2018-12-03/searchswagger.json [hunk 4]
```diff
@@ -0,0 +1,710 @@
...
+              },
+              "type": "array",
+              "uniqueItems": false
+            }
+          }
+        },
+        "tags": [
+          "Entities"
+        ],
+        "operationId": "ReadEntitesAsync",
```
Comment (nschonni) [inline specification/EnterpriseKnowledgeGraph/data-plane/Microsoft.EnterpriseKnowledgeGraph/stable/2018-12-03/searchswagger.json]:
```suggestion
        "operationId": "ReadEntitiesAsync",
```

File: specification/EnterpriseKnowledgeGraph/data-plane/Microsoft.EnterpriseKnowledgeGraph/stable/2018-12-03/searchswagger.json [hunk 5]
```diff
@@ -0,0 +1,710 @@
...
+        "summary": "ReadEntites"
```
Comment (nschonni) [inline specification/EnterpriseKnowledgeGraph/data-plane/Microsoft.EnterpriseKnowledgeGraph/stable/2018-12-03/searchswagger.json]:
```suggestion
        "summary": "Read entities"
```

**Comment by tengr** [comment]
All Errors Resolved. Please review and approve.

**Pull request closed without merging**